CMakeLists.txt

```diff
@@ -0,0 +1,256 @@
+header_copy(header13 IncludeBase IncludeCopy  ${CMAKE_SOURCE_DIR}/ohNet/OpenHome/Net/ControlPoint/Upnp ${CMAKE_CURRENT_BINARY_DIR}/Temp/OpenHome/Net/Private "${NetControlPointUpnp}")
+header_copy( header14 IncludeBase IncludeCopy  ${CMAKE_SOURCE_DIR}/ohNet/OpenHome ${CMAKE_CURRENT_BINARY_DIR}/Temp/OpenHome/Private "${PrivateHeaderFiles2}")
+header_copy(header15 IncludeBase IncludeCopy  ${CMAKE_SOURCE_DIR}/ohNet/OpenHome/Net ${CMAKE_CURRENT_BINARY_DIR}/Temp/OpenHome/Net/Core "${Net_pub}")
+header_copy(header16 IncludeBase IncludeCopy  ${CMAKE_SOURCE_DIR}/ohNet/OpenHome/Net/ControlPoint ${CMAKE_CURRENT_BINARY_DIR}/Temp/OpenHome/Net/Core "${NetControlPoint_pub}")
+header_copy(header17 IncludeBase IncludeCopy  ${CMAKE_SOURCE_DIR}/ohNet/Os ${CMAKE_CURRENT_BINARY_DIR}/Temp/OpenHome "${FolderOs}")
+header_copy(header18 IncludeBase IncludeCopy  ${CMAKE_SOURCE_DIR}/ohNet/OpenHome/Net/Bindings/C ${CMAKE_CURRENT_BINARY_DIR}/Temp/OpenHome/Net/C "${NetBindingsC}")
+header_copy(header19 IncludeBase IncludeCopy  ${CMAKE_SOURCE_DIR}/ohNet/OpenHome/Shell ${CMAKE_CURRENT_BINARY_DIR}/Temp/OpenHome/Private "${Shell_prv}")
+header_copy(header20 IncludeBase IncludeCopy  ${CMAKE_SOURCE_DIR}/ohNet/OpenHome/Net/Device ${CMAKE_CURRENT_BINARY_DIR}/Temp/OpenHome/Net/Private "${NetDevice}")
+header_copy(header21 IncludeBase IncludeCopy  ${CMAKE_SOURCE_DIR}/ohNet/OpenHome/Net/ControlPoint/Lpec ${CMAKE_CURRENT_BINARY_DIR}/Temp/OpenHome/Net/Private "${NetControlPointLpec_prv}")
+header_copy(header22 IncludeBase IncludeCopy  ${CMAKE_SOURCE_DIR}/ohNet/OpenHome/Net/Device ${CMAKE_CURRENT_BINARY_DIR}/Temp/OpenHome/Net/Core "${NetDevice_core}")
+header_copy(header23 IncludeBase IncludeCopy  ${CMAKE_SOURCE_DIR}/ohNet/OpenHome/Net/Device/Lpec ${CMAKE_CURRENT_BINARY_DIR}/Temp/OpenHome/Net/Private "${NetDeviceLpec_prv}")
+header_copy(header24 IncludeBase IncludeCopy  ${CMAKE_SOURCE_DIR}/ohNet/OpenHome/Net/Device ${CMAKE_CURRENT_BINARY_DIR}/Temp/OpenHome/Net/Cpp "${NetDevice_cpp}")
+header_copy(header25 IncludeBase IncludeCopy  ${CMAKE_SOURCE_DIR}/ohNet/OpenHome/Net/Device/Upnp ${CMAKE_CURRENT_BINARY_DIR}/Temp/OpenHome/Net/Private "${NetDeviceUpnp}")
+header_copy(header26 IncludeBase IncludeCopy  ${CMAKE_SOURCE_DIR}/ohNet/OpenHome/Net/Device/Providers ${CMAKE_CURRENT_BINARY_DIR}/Temp/OpenHome/Net/Core "${NetDeviceProviders}")
+header_copy(header27 IncludeBase IncludeCopy  ${CMAKE_SOURCE_DIR}/ohNet/OpenHome/Net/Device/Bonjour/mDNSCore ${CMAKE_CURRENT_BINARY_DIR}/Temp/OpenHome/Net/Private "${NetDeviceBonjourmDNSCore}")
+message( "Creating files in " ${CMAKE_CURRENT_BINARY_DIR}/Temp/OpenHome )
+target_include_directories( ohNet SYSTEM PRIVATE ${CMAKE_CURRENT_BINARY_DIR}/Temp )
+set( GENERATED_INCLUDES ${CMAKE_CURRENT_BINARY_DIR}/Temp PARENT_SCOPE )
+target_include_directories( ohNet SYSTEM PRIVATE ${CMAKE_SOURCE_DIR}/ohNet/OpenHome/Net/Bindings/Android/jni )
+if( ANDROID_ENDIAN )
```

Comment:
Not sure the value here, as android is completely little endian - though mostly by convention, not by edict